src/cli.cpp

```diff
@@ -313,9 +360,14 @@ int GetOpt(int argc, char *argv[], const std::vector<Option> &opts) {
 
     optind = 1;
     int opt;
-    while ((opt = getopt(argc + 1, argv - 1, optstring.c_str())) != -1) {
+    vector<string> mutableBuffer = Arguments;
+    vector<char*> rawArgs;
+    rawArgs.reserve(mutableBuffer.size());
+    for (auto & arg : mutableBuffer)
+        rawArgs.push_back(&arg[0]);
+    while ((opt = getopt(rawArgs.size(), &rawArgs[0], optstring.c_str())) != -1) {
```

Comment:
Here I changed old code behavior and skip previous (-1) command argument.
As I can see, usualy it was just a program name (was added to argument list in ICmd::RunCmd) and getopt will always skip it.
Am I right?